.eslintrc

```diff
@@ -69,6 +69,7 @@
 
     /* TODO | DISABLED TEMPORARILY */
     "react/sort-comp": 0,
+    "jsx-a11y/label-has-for": 0,
```

Comment:
`label-has-for` has been added to the TODO to complete after the airbnb plugin is successfully upgraded to the latest version. This rule will be replaced in the airbnb plugin  with `[label-has-associated-control](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/label-has-associated-control.md)`, therefore, we can't fix the 65 existing errors until airbnb is upgraded. It's been disabled temporarily in the meantime.

Reply:
I have verified that this is fixed automatically after the upgrade.